.github/workflows/main-auto-deploy.yml

```diff
@@ -0,0 +1,27 @@
+name: Subgraph Main Deployment
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+    types:
+      - closed
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - run: yarn
+      - run: yarn codegen
+      - run: yarn prepare:optimism
+      - run: yarn prepare:polygon
+      - run: yarn build:optimism && yarn deploy:optimism
+      - run: yarn build:polygon && yarn deploy:polygon
```

Comment:
We're missing mainnet here I blv

Reply:
ok, i didnt see a config for it in the code

Reply:
oh weird! I think we missed it - damn. Here's the SG link for it
https://thegraph.com/hosted-service/subgraph/otterspace-xyz/badges-mainnet